src/robot/model/body.py

```diff
@@ -95,22 +106,22 @@ def to_dict(self) -> DataDict:
         raise NotImplementedError
 
 
-class BaseBody(ItemList[BodyItem]):
+class BaseBody(ItemList[BodyItem], Generic[KW, F, W, I, T, R, C, B, M, E]):
     """Base class for Body and Branches objects."""
     __slots__ = []
     # Set using 'BaseBody.register' when these classes are created.
 
     if TYPE_CHECKING:
-        keyword_class = Keyword
-        for_class = For
-        while_class = While
-        if_class = If
-        try_class = Try
-        return_class = Return
-        continue_class = Continue
-        break_class = Break
-        message_class = Message
-        error_class = Error
+        keyword_class: Type[KW] = KnownAtRuntime
+        for_class: Type[F] = KnownAtRuntime
+        while_class: Type[W] = KnownAtRuntime
+        if_class: Type[I] = KnownAtRuntime
+        try_class: Type[T] = KnownAtRuntime
+        return_class: Type[R] = KnownAtRuntime
+        continue_class: Type[C] = KnownAtRuntime
+        break_class: Type[B] = KnownAtRuntime
+        message_class: Type[M] = KnownAtRuntime
+        error_class: Type[E] = KnownAtRuntime
     else:
```

Comment:
Is `if TYPE_CHECKING` still needed here? Isn't `Type[X] = KnownAtRuntime` always available?